metals/src/main/scala/scala/meta/internal/metals/BuildTargets.scala

```diff
@@ -51,6 +51,8 @@ final class BuildTargets() {
     ConcurrentHashSet.empty[AbsolutePath]
   private val buildTargetInference =
     new ConcurrentLinkedQueue[AbsolutePath => Seq[BuildTargetIdentifier]]()
+  // if workspace contains symlinks, original source items are kept here and source items dealiased
```

Comment:
you only call on this collection `.dealias.toNIO`. You can already keep dealiased path here then a `find` in `originalInverseSourceItem` will not have to do it every time.

Reply:
ahh no ok I get it, ignore :D